src/main/java/net/helpscout/api/cbo/WebhookEventType.java

```diff
@@ -0,0 +1,86 @@
+package net.helpscout.api.cbo;
+
+/**
+ * @Author: ivan
+ * Date: 23.07.16
+ * Time: 0:28
+ */
+public enum WebhookEventType {
+
+    Unknown(0, "unknown"),
+    Assigned(1, "convo.assigned"),
+    Created(2, "convo.created"),
+    Deleted(3, "convo.deleted"),
+    Merged(4, "convo.merged"),
+    Moved(5, "convo.moved"),
+    StatusUpdated(6, "convo.status"),
+    TagsUpdated(7, "convo.tags"),
+    CustomerReplyCreated(8, "convo.customer.reply.created"),
+    AgentReplyCreated(9, "convo.agent.reply.create"),
+    NoteCreated(10, "convo.note.created"),
+    CustomerCreated(11, "customer.created"),
+    RatingReceived(12, "satisfaction.ratings"),
+    TestEvent(13, "helpscout.test");
+
+    private final int value;
+    private final String label;
+
+    private WebhookEventType(int value, String label) {
```

Comment:
You could use `@AllArgsConstructor` and `@Getter` and remove the constructor and getters.